### ISSUE TITLE
[Bug] French text in link preview for google links

### ISSUE THREAD

**Issue by tprudentova** [linagora/twake-on-matrix#1382]
1. Open some chat 
2. Paste and send a google link 
3. Text in preview is in French, though my app and system language is English, and my location is Russia 

<img width="788" alt="Screenshot 2024-01-22 at 13 27 20" src="https://github.com/linagora/twake-on-matrix/assets/97684890/c8b7648e-d17d-45b5-bb62-2da8313a920c">

**Comment by guimard** [comment]
@hoangdat : is it related to the `url_preview_enabled` you make me change in Matrix/Synapse ?

**Comment by hoangdat** [comment]
@guimard yes, from the client side, we can not request other parameters except: `url`
https://spec.matrix.org/v1.6/client-server-api/#get_matrixmediav3preview_url
